greenplumpython/dataframe.py

```diff
@@ -1183,4 +1207,9 @@ def from_columns(cls, columns: Dict[str, Iterable[Any]], db: Database) -> "DataF
         columns_string = ",".join(
             [f'unnest({_serialize_to_expr(list(v), db=db)}) AS "{k}"' for k, v in columns.items()]
         )
-        return DataFrame(f"SELECT {columns_string}", db=db)
+        return cls(f"SELECT {columns_string}", db=db)
+
+    # Add interface here for language servers.
+    def embedding(self) -> "Embedding":
+        """Allow user to process vector operators thanks to :class:`~experimental.embedding.Embedding`."""
```

Comment:
Should we remind users that they should do `import greenplumpython.experimental.embedding` otherwise they will get a `NotImplementedError` from this function when they try to do `t.embedding().create_index(column, model)`